rosdep/python.yaml

```diff
@@ -2628,6 +2628,7 @@ python-numpy:
     lucid: [python-numpy]
     maverick: [python-numpy]
     natty: [python-numpy]
+    noetic: [python-numpy]
```

Comment:
Noetic is the name of the ROS distro. This would need to be the Ubuntu distro `focal`.